src/routes/Dashboard/containers/DashboardContainer.js

```diff
@@ -7,6 +7,7 @@ const mapActionCreators = {
 const mapStateToProps = (state) => ({
   demoName: state.demoSession.name || 'loading...',
   dbdata: state.dashboard,
```

Comment:
As we start working with the data and know how it should look in the UI we need to normalize this and validate the keys of the object (dbdata was just used initially to blast the entire json call to the screen)